app/javascript/components/upload/MetadataStep.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { useEffect, useContext } from 'react'
 import metadataExplainerImage from '~/../assets/images/metadata-convention-explainer.jpg'
+import annDataMetadataExplainerImage from '~/../assets/images/metadata-convention-explainer-anndata.jpg'
```

Comment:
Nit: non-photographic raster images tend to use WEBP or PNG instead of JPG, because the former can be lossless.  JPG is more common for photos, where lossy compression makes a lot of sense.  

The image looks good and this is definitely not a blocker, especially since the other image uses JPG.  But WEBP or PNG might be better if we ever recreate such images from scratch and want very crisp graphics.

Reply:
Good point, @eweitz. I only generated jpg because the previous image was in that format.
I loaded [mc_compliant_metadata_image_full.psd ](https://github.com/broadinstitute/single_cell_portal/blob/master/site_figures/metadata_file/mc_compliant_metadata_image_full.psd) into GIMP and generated a PNG
![mc_compliant_metadata_image_full](https://github.com/user-attachments/assets/9e16962f-4d57-4447-9548-bf1ab217e0b8)
and a PNG for the AnnData version
![anndata_compliant_metadata_image_full_edited](https://github.com/user-attachments/assets/c62ec1e3-38dc-4b75-a8c3-7786b0ededd8)
Are there any gotchas where a round trip through GIMP might have messed with the resolution of the image (that would not be detectable on my screen).

@bistline Not entirely sure if Github retains the full resolution image this way. Let me know if I should get the files to you another way.

Reply:
Thanks for the comparison!  The attached WEBP is somewhat smaller in byte size and looks equivalent in quality compared to the PNG.  GitHub retains the raw file.  

<img width="1472" alt="Compare_WEBP_PNG_JPG__2024-08-19" src="https://github.com/user-attachments/assets/8d303bce-f379-46b2-89e8-db2e3f92e853">

GitHub comments seem to prevent WEBP attachments, but WEBP files are allowed in GitHub repos.  Simply remove the .txt extension from the attached file to get original WEBP.  [WEBP support](https://caniuse.com/webp) is effectively universal.